Display content for job items.

jaden opened the pull request:
I'd like a cleaner way to toggle the 'No comments yet'
message, but I don't know how (or if it's possible) to access
Item.isJob() from ItemView.

Commit:
Display content for job items.

I'd like a cleaner way to toggle the 'No comments yet'
message, but I don't know how (or if it's possible) to access
Item.isJob() from ItemView.

yyx990803 commented on src/components/Item.vue:
```diff
@@ -17,6 +17,7 @@
         | <a :href="'#/item/' + item.id">{{item.descendants}} {{item.descendants | pluralize 'comment'}}</a>
       </span>
     </p>
+    <p class="jobtext" v-show="isJob" v-html="item.text"></p>
```
Let's use `v-if` here since this is not going to change at runtime

jaden replied:
Ah, I wasn't quite clear on the distinction. Thanks for the clarification.

yyx990803 replied:
There's some more details here: http://vuejs.org/guide/conditional.html#v-if_vs-_v-show

jaden commented:
I also just realized I had been testing the individual item listings, but the news listing shows the full content of a job listing.

Commit:
Show job text in ItemView rather than Item.

I think this makes it cleaner, and job text isn't shown in the NewsView.

Also, using v-if rather than v-show since item type won't change at
runtime.

Commit:
Show text for any item with text.

Show the text of any item that has it.

Jobs don't have comments, so don't show the 'No comments yet' message.
